common/reviews/api/terminal.api.md

```diff
@@ -82,6 +219,11 @@ export interface ITextRewriterTransformOptions extends ITerminalTransformOptions
     textRewriters?: TextRewriter[];
 }
 
+// @beta (undocumented)
+export interface IWriteOptions {
```

Comment:
Suggest to call this `ITerminalWriteOptions`; in the context of computer science `IWriteOptions` is pretty vague